sections/plate_notation.tex

```diff
@@ -77,43 +39,23 @@ \subsection{Author-Topic \gls{lda} and Category-Topic \gls{lda}}
 The second and third layers will be constructed based on the entries of the first two positions in our taxonomy metadata, meaning there will be one node for each unique sub-taxonomy entry that is in the first or second position in the taxonomy sequence (e.g. "STEDER" and "Danmark", which is taken from "STEDER/Danmark/Aalborg", but not "Aalborg" since it is in the third position).
-We only use the first two layers for this, as these are among the most informative, and because introducing even more layers would slow down the training exponentially, since the probability of all possible combinations of topics needs to be sampled for every word during training. \vejleder[inline]{can you back this up with experiments or hard evidence?}
+We only use the first two layers for this, as these are among the most informative, and because introducing even more layers would slow down the training significantly, since the probability of all possible combinations of topics needs to be sampled for every word during training.
+From our experiments, the training time for 50 epochs, increases from 12 hours to 130 hours between four and five level pachinko.
 The fourth layer consists of $K = 90$ 'blank' topics, as with the other models we present in this paper.
 This layer is added so that the model can construct its own topics based on the higher-level topics learned from our taxonomy metadata.
-This \gls{dag} structure is visualized in \autoref{fig:pachinko_dag}. \vejleder[inline]{Figure 4 shows both the original and your adaption? rephrase}
-
-The generative process for each document $d$ in \gls{pam} consists of the following steps, as described by \citet{li2006pachinko}:
-\begin{enumerate}
-	\item Sample $\theta_{t_1}^{(d)}, \theta_{t_2}^{(d)}, \dots, \theta_{t_s}^{(d)}$ from $g_1(\alpha_1), g_2(\alpha_2), \dots, \newline g_s(\alpha_s)$, where $\theta_{t_i}^{(d)}$ is a multinomial distribution of topic $t_i$ over its children.
-	\item For each word $w \in d$,
-	\begin{itemize}
-		\item Sample a topic path $\mathbf{z}_w$ of length $L_w:~< z_{w1}, z_{w2},\newline \dots, z_{wL_w} >$. $z_{w1}$ is always the root and $z_{w1}$ through $z_{wL_w}$ are topic nodes in $T$. $z_{wi}$ is a child of $z_{w(i-1)}$ and it is sampled according to the multinomial distribution $\theta_{z_{wL_w}}^{(d)}$.
-		\item Sample word $w$ from $\theta_{z_{wL_w}}^{(d)}$.
-	\end{itemize}
-\end{enumerate}
-
-$T = {t_1, t_2, \dots, t_s}$ is the set of topics in the \gls{pam}. 
-Each topic $t_i$ is associated with a Dirichlet distribution $g_i(\alpha_i)$ based on a vector $\alpha_i$ which has the same dimension as the number of children in $t_i$.
-While it is possible to use different $\alpha$ values for each topic, as shown here, we found through experimentation that using the same $\alpha$ value for all layers still provided good results.
-An illustration of the plate notation for \gls{pam} can be found in \autoref{fig:pachinko}.
 
 We use Gibbs sampling for performing inference.
```

Comment:
Det kan være, som jeg forstod det fra Thomas, at det vil give mere mening at have denne Gibbs sampling beskrivelse sammen med den i bilaget, og så bare her referere til appendix. Så sparer vi også lidt plads.

Reply:
for min skyld gerne

Reply:
Jeg kan se at "Pachinko Implementation" i appendix også beskrive skridtene i Pachinkos Gibbs sampling, så det er også en mulighed at flytte den derned. Ved dog ikke hvad vil passe bedst.